mindbender/tools/projectmanager/lib.py

```diff
@@ -0,0 +1,72 @@
+"""Utility script for updating database with configuration files
+
+Until assets are created entirely in the database, this script
+provides a bridge between the file-based project inventory and configuration.
+
+- Migrating an old project:
+    $ python -m mindbender.inventory --extract --silo-parent=f02_prod
+    $ python -m mindbender.inventory --upload
+
+- Managing an existing project:
+    1. Run `python -m mindbender.inventory --load`
+    2. Update the .inventory.toml or .config.toml
+    3. Run `python -m mindbender.inventory --save`
+
+"""
+
+
+from mindbender import schema, io
+
+
+def create_asset(data):
+    """Create asset
+
+    Requires:
+        {"name": "uniquecode",
+         "label": "Nice readable name",
```

Comment:
I'm not sure `label` is actually required? I don't believe it is; the docstring might be confusing.